client/apps/game/src/ui/modules/ws-chat/chat.tsx

```
@@ -364,6 +364,19 @@ function ChatModule() {
     };
   }, [chatClient]);
 
+  // Automatically join all available rooms to receive notifications
+  useEffect(() => {
+    if (chatClient && chatClient.socket && chatClient.socket.connected && availableRooms.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)